src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -167,6 +167,7 @@ public async Task SendStreamLimitRequestsConcurrently_Succeeds(int streamLimit)
         [InlineData(10)]
         [InlineData(100)]
         [InlineData(1000)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/55901")]
```

Comment:
The issue says only about limit=10 -- is it also failing for other limits?

Reply:
That's on purpose here from my experience with how mocks behaved. If I disabled one, it started to fail on the next input value. So I'd rather be on the safe side and disable them all at once.

Reply:
I see. If it's affecting others a lot I guess it's better better to be on the safe side. However, I was sure that all the inputs are run even if one of them fails...

Reply:
Yes they do run. 
What I'll say know will sound super proffesionally :smile: but what happened with mocks was that always the first call failed (timed out) and the subsequent passed, no matter what you disabled/enabled. They shouldn't have affect each other, since the connection is always new etc., but that's what I was seeing and eventually gave up investigating.

Reply:
Should we disable it only on mock if there is way how to detect it?

Reply:
Mocks has been disabled for a while, now we have a failure on msquic (for which this PR is).
Since I had a bad experience with trying to disable just one input on mock (always leading to failure one step further), I'm preemptively disabling them all here. That's what this discussion was about.